docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,56 @@ tags:
-
-## Get started {#get-started}
-
-To get started, ensure you're a Temporal Cloud user.
-If you don't have an account with Temporal, **[sign up here](https://pages.temporal.io/get-started-with-cloud)**.
-
-### Prerequisites
-
-- A Cloud user account
-- Access to the [Temporal Cloud UI](https://cloud.temporal.io/) or Temporal Cloud CLI ([tcld](https://docs.temporal.io/cloud/tcld/))
-- [Enable access to API Keys](#manage-api-keys)
+API Keys support 2 main use cases:
 
-## Use API Keys {#use-api-keys}
+- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, namespaces, certificates, and user identities.
+  - API Keys for use in Cloud Operations Tools are in Public Preview
+- _**Namespace Authentication**_: API Keys can be used as the Namespace authentication mechanism for executing and managing workflows from the SDK and Temporal CLI as an alternative to using mTLS-based authentication.
+  - API Keys for use in Namespace Authentication are in Pre Release
 
-API Keys can be used in the `tcld` or in the Cloud Ops API.
+API Keys are an opt-in feature that must be enabled by a Global Administrator for use. For instructions on API Key Management for your organization, see [Global Administrator API Key Management](#ga-manage-api-keys).
 
-**How can I use the API Key with tcld?**
+API Keys are supported for Users and Service Accounts.
 
-To use your API Key with tcld, use one of the following methods:
+- Users have the autonomy to create, delete, and update access to their API Keys through the Cloud UI or tcld. For broader oversight, Global Administrators can manage user's API Keys using the same interfaces.
+- Global Administrators can create, delete, and update access to API Keys for Service Accounts.
 
-- `--api-key` flag
-- `TEMPORAL_CLOUD_API_KEY` environment variable
+API Keys can be used as an authentication mechanism with:
 
-**How can I use the API Key with Cloud Ops API?**
+- [Temporal CLI v 0.12 or higher](/cli)
+- [SDKs (latest recommended)](/develop)
+- [Temporal Cloud CLI tcld](/production-deployment/cloud/tcld/index.mdx)
+- [The Cloud Operations API](/production-deployment/cloud/operation-api.mdx)
+- [Temporalʼs Terraform Provider](/production-deployment/cloud/terraform-provider)
 
-To use your API Key with the [Cloud Ops API](/ops), securely pass your API Key in your Cloud Ops API client.
+## Get started {#get-started}
 
-For a complete example, see [Cloud Samples in Go](https://github.com/temporalio/cloud-samples-go/blob/1dd4254b6ed1937e361005c0144410e72b8a5542/client/api/apikey.go).
+### Prerequisites
 
-## Manage API Keys {#manage-api-keys}
+- Global Adinistrator should [Enable access to API Keys](#manage-api-keys)
```

Comment:
```suggestion
- Global Administrator should [Enable access to API Keys](#manage-api-keys)
```

Reply:
This is missing a pronoun before Global.

Same issue with Global Administrator. De-cap or add to linter as official capped term.

Make sure "Enable access to API keys" is either fixed for capitalization or adjust wording to indicate that this is verbatim so it can be found on a web page, normally by "selecting 'Enable access to API Keys' option on the Web interface" or similar.